task/oci-copy/0.1/oci-copy.yaml

```
@@ -99,36 +99,75 @@ spec:
 
         set -u
 
+        echo "Selecting auth for $IMAGE"
+        select-oci-auth $IMAGE > auth.json
+
+        echo "Extracting artifact_type"
+        ARTIFACT_TYPE=$(cat "$(pwd)/source/$OCI_COPY_FILE" | yq '.artifact_type')
+
+        REPO=$(echo ${IMAGE} | awk -F ':' '{print $1}')
```

In https://github.com/konflux-ci/build-definitions/pull/1129/files and https://github.com/konflux-ci/build-definitions/blob/c316db02a8fe4f67ba7fd36c021e367ebad0b4fc/task/rpm-ostree/0.1/rpm-ostree.yaml#L264-L265 we use this, which theoretically works if the IMAGE reference contains a port number
```suggestion
        REPO=${IMAGE%:*}
```